pr-statistics-spent-time.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+
+# PARAMETERS ----------------------------
+GITHUB_REPO="status-im/status-dev-cli"
+DEBUG=1
+
+# PLATFORM ----------------------------
+case "$OSTYPE" in
+  darwin*)  OS="DARWIN" ;;
+  *)        OS="LINUX" ;;
+esac
+
+# FUNCTIONS ----------------------------
+
+github_pr() {
+  repo="$1"
+  list_spent_time=""
+  list_dt_closed=""
+  list_dt_created=""
+
+	last_page=$(curl -i -sL "https://api.github.com/repos/$repo/pulls?state=closed" | grep rel=\"last\" | cut -d "," -f 2 | cut -d "=" -f 3 | cut -d ">" -f 1)
+  [ "$last_page" = "" ] && last_page=1
+  [ "$DEBUG" = "1" ] && echo "DEBUG -- url is https://api.github.com/repos/$repo/pulls?state=closed -- nb page : $last_page" >&2
+  for i in $(seq 1 $last_page); do
+    [ "$DEBUG" = "1" ] && echo "DEBUG -- downloading page $i ..." >&2
+    json="$(curl -sL https://api.github.com/repos/$repo/pulls?state=closed&page=$i)"
+
+    [ "$DEBUG" = "1" ] && echo "DEBUG -- page $i downloaded" >&2
+    list_dt_closed="$list_dt_closed $(echo "$json" | grep -B2 "closed_at" | grep -A2 "created_at" | grep "closed_at" | tr -d ' ' | cut -d "\"" -f 4)"
+		list_dt_created="$list_dt_created $(echo "$json" | grep -B2 "closed_at" | grep -A2 "created_at" | grep "created_at" | tr -d ' ' | cut -d "\"" -f 4)"
+
+    len_list_dt="$(echo $list_dt_created | wc -w)"
+
+    for k in $(seq 1 $len_list_dt); do
+      dt_crea="$(echo $list_dt_created | cut -d " " -f $k)"
+      [ "$OS" = "DARWIN" ] && dt_crea="$(date -j -u -f '%Y-%m-%dT%H:%M:%SZ' "$dt_crea" +%s)" \
+        || dt_crea="$(date -d "$dt_crea" +%s)"
+
+      dt_closed="$(echo $list_dt_closed | cut -d " " -f $k)"
+      [ "$OS" = "DARWIN" ] && dt_closed="$(date -j -u -f '%Y-%m-%dT%H:%M:%SZ' "$dt_closed" +%s)" \
+        || dt_closed="$(date -d "$dt_closed" +%s)"
+
+      (( spent_time = dt_closed - dt_crea ))
+      list_spent_time="$list_spent_time $spent_time"
+    done
+
+  done
+
+  echo "$list_spent_time" | tr ' ' '\n' | sort -n | tr ' ' '\n'
+}
+
+median_list() {
+  list="$@"
+  len_list="$#"
+  median=""
+
+  (( middle_list = len_list / 2 ))
+  (( middle_list_next = middle_list + 1 ))
+
+  j=1
+  for l in $list; do
+    [ $j -eq $middle_list ] && median=$l
+    if [ $j -eq $middle_list_next ]; then
+      median=$(echo "scale=2; ($median + $l)/2" | bc -l)
+      break
+    fi
+    (( j = j + 1 ))
+  done
+  echo "$median"
+}
+
+five_nb_summary() {
+  list="$@"
+  len_list="$#"
+  sample_minimum=""
+  lower_quartile=""
+  median=""
+  upper_quartile=""
+  sample_maximum=""
+
+  # NOTE list is already sorted asc
+  sample_minimum=$(echo $list | cut -d " " -f 1)
+  sample_maximum=$(echo $list | cut -d " " -f $len_list)
+
+  median="$(median_list $list)"
+
+  i=1
+  for l in $list; do
```

Comment:
Can we format this by columns? So all first columns take up X width and the numbers are aligned

Reply:
done in commit https://github.com/status-im/status-react/pull/2893/commits/0b287d778877586a7249552f78af30f2223027ac